tools/c7n_gcp/tests/data/flights/gke-cluster-nodepool-query/get-v1-projects-cloud-custodian-locations-us-central1-a-clusters-standard-cluster-1-nodePools_1.json

```diff
@@ -0,0 +1,120 @@
+
+{
+  "body": {
+    "nodeIpv4CidrSize": 24,
+    "networkPolicy": {},
+    "masterAuthorizedNetworksConfig": {},
+    "addonsConfig": {
+      "httpLoadBalancing": {},
+      "networkPolicyConfig": {
+        "disabled": true
+      },
+      "kubernetesDashboard": {
+        "disabled": true
+      }
+    },
+    "locations": [
+      "us-central1-a"
+    ],
+    "legacyAbac": {},
+    "defaultMaxPodsConstraint": {
+      "maxPodsPerNode": "110"
+    },
+    "network": "default",
+    "loggingService": "logging.googleapis.com",
+    "instanceGroupUrls": [
+      "https://www.googleapis.com/compute/v1/projects/cloud-custodian/zones/us-central1-a/instanceGroupManagers/gke-standard-cluster-1-default-pool-6bd82ab6-grp"
+    ],
+    "networkConfig": {
+      "subnetwork": "projects/cloud-custodian/regions/us-central1/subnetworks/default",
+      "network": "projects/cloud-custodian/global/networks/default"
+    },
+    "ipAllocationPolicy": {},
+    "zone": "us-central1-a",
+    "servicesIpv4Cidr": "10.11.240.0/20",
+    "status": "RUNNING",
+    "currentNodeVersion": "1.11.8-gke.6",
+    "labelFingerprint": "a9dc16a7",
+    "currentMasterVersion": "1.11.8-gke.6",
+    "masterAuth": {
+      "username": "admin",
+      "password": "IWqLAzJg7bTJYP5M",
```

Comment:
this is kinda of shocking, and a security issue for gcp

Reply:
Updated

Reply:
the fact that master passwords are being logged is something that needs to be flagged to gcp folks at google as a security issue.

Reply:
oh this isn't even logging isn't on api trail, this is part of a get request for a cluster.. hmmm okay. so read access to a project means root access to your cluster.. wtf.

Reply:
This section has been deleted in a mock.

Reply:
sorry the wtf wasn't in ref to the pr, more to just that this passwords were being returned from read apis at all from the gcp api. i went ahead and talked to some gcp folks, and was told this was a legacy auth mechanism for older clusters that is now deprecated.